package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-conversio",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

```suggestion
  "version": "4.0.0",
```

A bit ambiguous I know, but adding all these rules is almost surely a breaking change 🤔